<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CombatantWidget</class>
 <widget class="QWidget" name="CombatantWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CombatantWidget</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>267</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QSpinBox" name="curHPSpinBox"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="curHPLabel">
      <property name="text">
       <string>Current Hit Points:</string>
      </property>
      <property name="buddy">
       <cstring>curHPSpinBox</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="Line" name="playerLine">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="playerLabel">
      <property name="text">
       <string>Player</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QToolButton" name="editButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="4">
     <widget class="QTextEdit" name="notesEdit"/>
    </item>
    <item row="6" column="0" colspan="4">
     <widget class="QLabel" name="notesLabel">
      <property name="text">
       <string>Notes:</string>
      </property>
      <property name="buddy">
       <cstring>notesEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="4">
     <widget class="QLabel" name="otherInfo">
      <property name="text">
       <string>&lt;no info available&gt;</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="Line" name="hpLine">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="Line" name="titleLine">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QLabel" name="maxHPLabel">
      <property name="text">
       <string>Maximum Hit Points:</string>
      </property>
      <property name="buddy">
       <cstring>maxHPEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QLineEdit" name="maxHPEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1" rowspan="2">
     <widget class="Line" name="hpSeparatorLine">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
